app/src/main/java/com/scrumbums/donationboi/controllers/LoginActivity.java

```diff
@@ -339,8 +332,7 @@ protected void onPostExecute(final Boolean success) {
             if (success) {
                 finish();
                 startActivity(new Intent(LoginActivity.this, ApplicationActivity.class));
-
-            } else {
+            } else if (loginResult == 0){
```

Comment:
use `!loginResult` pls

Reply:
loginResult is an integer, to differentiate between the following results:
- Success
- Account exists but password is incorrect
- Account has not been registered

idk how login works, so i didn't want to replace success outright. please advise since you probably know more about login activities than I

Reply:
Ah, I see.  That's fine then